app/messaging/message-base.js

```diff
@@ -16,7 +16,11 @@ class MessageBase {
     } else if (this.config.connectionString) {
       this.sbClient = new ServiceBusClient(this.config.connectionString)
     } else {
-      this.sbClient = new ServiceBusClient(`Endpoint=sb://${this.config.host}/;SharedAccessKeyName=${this.config.username};SharedAccessKey=${this.config.password}`)
+      var useDevelopmentEmulator=false;
+      if (this.config.useEmulator) {
+        var useDevelopmentEmulator=this.config.useEmulator;
+      }
+      this.sbClient = new ServiceBusClient(`Endpoint=sb://${this.config.host}/;SharedAccessKeyName=${this.config.username};SharedAccessKey=${this.config.password};UseDevelopmentEmulator=${useDevelopmentEmulator}`);
```

Comment:
Would suggest tweaking so that the full argument is omitted if not using the emulator.

eg

```
let useDevelopmentEmulator

if (this.config.useEmulator) {
  useDevelopmentEmulator = ';UseDevelopmentEmulator=true'
}

this.sbClient = new ServiceBusClient(`Endpoint=sb://${this.config.host}/;SharedAccessKeyName=${this.config.username};SharedAccessKey=${this.config.password}${useDevelopmentEmulator}`
```

Then there's no risk of a side effect in any existing application from `UseDevelopmentEmulator=false`